### ISSUE TITLE
Can't add specific novel on wuxiaworld.site, AttributeError: 'NoneType' object has no attribute 'get_text'

### ISSUE THREAD

**Issue by chrisric** [JimmXinu/FanFicFare#657]
Hi,

I get the below error/s when trying to add "https://wuxiaworld.site/novel/in-a-different-world-with-a-smartphone-novel", I have other wuxiaworld.site novels that work fine.

Thanks for the help.

-Chris

```
FFF: DEBUG: 2021-02-14 10:19:17,141: calibre_plugins.fanficfare_plugin.fff_plugin(1099): FanFicFare v3.29.0
FFF: DEBUG: 2021-02-14 10:19:17,256: calibre_plugins.fanficfare_plugin.fff_plugin(1220): prep_download_loop url:https://wuxiaworld.site/novel/in-a-different-world-with-a-smartphone-novel
FFF: DEBUG: 2021-02-14 10:19:17,282: calibre_plugins.fanficfare_plugin.fanficfare.adapters.adapter_wuxiaworldsite(82): URL: https://wuxiaworld.site/novel/in-a-different-world-with-a-smartphone-novel
FFF: DEBUG: 2021-02-14 10:19:17,283: calibre_plugins.fanficfare_plugin.fanficfare.configurable(1209): #####################################
pagecache(GET) MISS: https://wuxiaworld.site/novel/in-a-different-world-with-a-smartphone-novel
FFF: DEBUG: 2021-02-14 10:19:20,074: calibre_plugins.fanficfare_plugin.fanficfare.configurable(1031): Encoding:utf8
FFF: ERROR: 2021-02-14 10:19:21,227: calibre_plugins.fanficfare_plugin.dialogs(723): Exception: {u'site': 'wuxiaworld.site', u'comments': u'', u'collision': u'Update EPUB if New Chapters', u'series': u'', u'pubdate': None, u'title': u'Unknown', u'listorder': 0, u'status': u'Error', u'url': u'https://wuxiaworld.site/novel/in-a-different-world-with-a-smartphone-novel', u'end': None, u'good': False, u'calibre_id': None, u'author_sort': [u'Unknown'], u'publisher': None, u'showerror': True, u'added': False, u'comment': u"'NoneType' object has no attribute 'get_text'", u'begin': None, u'author': [u'Unknown'], u'uniqueurl': u'https://wuxiaworld.site/novel/in-a-different-world-with-a-smartphone-novel[None-None]'}:'NoneType' object has no attribute 'get_text'
Traceback (most recent call last):
  File "calibre_plugins.fanficfare_plugin.dialogs", line 710, in do_loop
  File "calibre_plugins.fanficfare_plugin.fff_plugin", line 1305, in prep_download_loop
  File "calibre_plugins.fanficfare_plugin.fff_plugin", line 1185, in get_story_metadata_only
  File "calibre_plugins.fanficfare_plugin.fanficfare.adapters.base_adapter", line 298, in getStoryMetadataOnly
  File "calibre_plugins.fanficfare_plugin.fanficfare.adapters.base_adapter", line 394, in doExtractChapterUrlsAndMetadata
  File "calibre_plugins.fanficfare_plugin.fanficfare.adapters.adapter_wuxiaworldsite", line 95, in extractChapterUrlsAndMetadata
AttributeError: 'NoneType' object has no attribute 'get_text'
```

**Comment by JimmXinu** [comment]
That particular story doesn't list an author.  It instead says "Author(s) Updating"

Which apparently is a thing, because I found [another one](https://wuxiaworld.site/novel/why-are-you-so-obsessed-with-rejecting-affection/) after clicking on only a very few stories.

I've posted a [test version plugin](https://www.mobileread.com/forums/showthread.php?p=3084025&postcount=2) that uses what it says "Updating" for the author.  If it was absent entirely, I'd use "Anonymous".

> **New Test Version Posted**
> 
> **2021-02-13**
> - Fix for 'Updating' (anonymous?) author in adapter_wuxiaworldsite. For #657
